src/context/filter-context.js

```diff
@@ -0,0 +1,34 @@
+import { createContext , useReducer , useContext} from "react";
+import { useProducts } from "./product-context";
+import { sortedData, FilterPriceData ,FilterCategoryData , FilterRatingData , ComposeAll} from '../utils/filterutils';
```

Comment:
Suggestion: barrel export can be use here

Reply:
as of now, this fine for me will update in future